src/python/WMCore/REST/Validation.py

```diff
@@ -26,22 +30,22 @@ def _check_rx(argname, val, custom_err = None):
 
 def _check_str(argname, val, rx, custom_err = None):
     """
-    convert unicode to str first to support cherrypy 3.2.2
+    convert unicode to str (bytes) first to support cherrypy 3.2.2
```

Comment:
this comment makes me wonder if we even need it... WMCore and DBS are using CP 17.x, CRAB is on 5.x...
Rhetorical comment though :)

Reply:
I added it to the infinite list of things to check after the migration is completed :)

Reply:
I decided that not to lose these comments about wollow up checks and investigations, I should add a section to the PR description at the top with the "Open question that should be addressed in follow up work". is this ok with you?

Reply:
Sure. I'm just not convinced that there will be any follow up once the PR gets merged. In that case, we better go with a GH issue if the open question hasn't been resolved.

Reply:
Should I open a github issue for this, too?

Reply:
I guess we can move forward without it, thanks!